etc/org.opencastproject.transcription.amberscript.AmberscriptTranscriptionService.cfg

```diff
@@ -8,6 +8,24 @@
 # Can be one of: nl, en, de, fr, da, sv, fi, no, es
 # language=en
 
+# Set to true if the transcription language should be taken from the dublincore catalog
+# If the dublincore catalog doesn't contain a language, the default process for determine
+# the language will be taken. Note, that the language field in the workflow operation
+# will be overwritten, in case the dublincore catalog has a language
+# Default=false
+#language.from.dublincore=false
+
+# Manual mapping for language codes
+# Here you can set which language spellings/codes from your system shall be
+# mapped to the corresponding amberscript language. In general the amberscript workflow
+# should figure it out by itself. Use this, if you have some special language strings and run into errors
+# You can map a language to the amberscript code with a Colon ':'
+# You can separate multiple mappings with a comma ','
+#example1=english:en
+#example2=german:de,french:fr,spanish:es
+#example3=SomeWeirdL4nguageC0de:en
```

Comment:
These examples are confusing.  Suggest something like
`#Examples:`
`# language.code.map=english:en`
`# language.code.map=german:de,french:fr,spanish:es`
`# language.code.map=SomeWeirdL4nguageC0de:en`

The goal here would be to ensure that it's obvious that the `example` part isn't supposed to be there, and that you can only specify `language.code.map` once.